src/components/Info/styles.styl

```diff
@@ -0,0 +1,24 @@
+@require 'settings/palette.styl'
```

Comment:
you can remove it. It's not necessary and I duplicate code https://github.com/cozy/cozy-ui/issues/762